config/initializers/discord_bot.rb

```diff
@@ -12,7 +12,7 @@ def bot
   private
 
   def token
-    token = Rails.application.credentials.discord.bot.token
+    @token ||= ENV["DISCORD_BOT_TOKEN"]
```

Comment:
Should be env || credentials, here and in other places.
So both approaches supported.

Reply:
I think we shouldn't be using credentials, since we won't be sharing a private key and any run of `Rails.application.credentials.discord.bot.token` will fail with nil method call.